### PR TITLE
Mini feed

### PR DESCRIPTION
**Pull request by nattydodd**
Hey Wes, 
Made more changes to the feed, only changed two things that would clash with yours:

-main logo: changed to new one that Ben wanted
-added an ID to the search bar so it wouldn't affect something on my code

Thanks!

### CODE REVIEW

File: app/assets/stylesheets/pages.scss [hunk 1]
```diff
@@ -824,7 +824,7 @@ body{
 
 
 
-input{
```
Comment (Henshall) [inline app/assets/stylesheets/pages.scss]: This messes up the input values on the front page

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -11,7 +11,7 @@
     <%current_user = nil  %>
     <div class="top_bar">
 
-      <%= image_tag 'logo.png', class: 'float_left logo fixed_left' %>
```
Comment (Henshall) [inline app/views/layouts/application.html.erb]: ben didnt like the globe logo I picked out for him? - He wanted it changed back?

Comment (Henshall) [inline app/views/layouts/application.html.erb]: ben didnt like the globe I added?

File: app/assets/stylesheets/pages.scss (hunk 1 above)
Comment (nattydodd) [inline app/assets/stylesheets/pages.scss]: I think I fixed this by adding those input values into the stylesheet using their ID/Class (look for #searchbar and .front styles on pages.scss)

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (nattydodd) [inline app/views/layouts/application.html.erb]: Ya he put the link to this logo in the docs and said he wanted it changed back

**Pull request merged**